pager.c

```diff
@@ -2926,6 +2926,47 @@ int mutt_pager(const char *banner, const char *fname, PagerFlags flags, struct P
         rd.topline = new_topline;
         break;
 
+// ----------------------------------------------------------------------------
+      case OP_PAGER_SKIP_HEADERS:
+        if (rd.has_types)
+        {
+          int dretval = 0;
+          int new_topline = rd.topline;
+ 
+          if (!IS_HEADER (rd.line_info[new_topline].type))
+          {
+            /* L10N:
+               Displayed if <skip-headers> is invoked in the pager, but we are
+               already past the headers
+            */
+            mutt_error(_("Already skipped past headers."));
```

Comment:
Mutt has a lot of **errors**.
If we can tell the user **why** their action failed (or what to do next time),
then I'd like to downgrade the `mutt_error()` to `mutt_warning()` or `mutt_message()`.